### DISCUSSION TITLE
No notification on absent artifact in artifact dir

### DISCUSSION THREAD

**Discussion by gtgray** [Triple-T/gradle-play-publisher#999] [Q&A] [answered]
Hello!
Please, tell me, how to adjust plugin to show notification/exception when artifact not found in predefined artifact directory?

**Comment by SUPERCILEX** [comment, marked as answer]
I believe it already throws an exception if nothing is found in the artifact directory. If not, you could always write a bit of code that checks this yourself: `if (!file("blah").exists()) ...`.

**Reply by gtgray** [reply]
It throws an exception on no artifact directory found, but unfortunately, it doesn't throw anything on absent artifact.
I launch "publishBundle" task with nothing in artifact directory and task returns success state and no error or any warning line.
Of course, I can check this case myself, but to be honest I expect that plugin that makes publishing of some artifact should say anything if that artifact has not been found

**Reply by SUPERCILEX** [reply]
Ah, yeah I think you're right. I created #1001 to take a look.
